templates/palletInfo.html

```diff
@@ -3,11 +3,13 @@
     Pallet Page
 {% endblock %}
 {% block content %}
+<br/>
+    <button type="button" class="btn btn-success btn-lg" onclick="goBack()">&#8592;Back</button>
```

Comment:
you shouldn't need a js function to do this, if you're passing the location id to the template then use the _{{ url_for(route, location_id=location.id) }}_